{"name": "test_change_to_invalid_username", "status": "passed", "attachments": [{"name": "invalid username error message", "source": "b5ae227c-4e69-4998-84cd-e302f89ff66b-attachment.png", "type": "image/png"}, {"name": "log", "source": "89b771e2-d40f-4197-aca2-23e21b1bd17d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "30e8e1e6-6175-4823-bf84-59a8ac677aca-attachment.txt", "type": "text/plain"}], "start": 1723157708494, "stop": 1723157736472, "uuid": "b48bcd12-273a-413f-bb57-83a148282e92", "historyId": "dc0a9cd52b3898118b858a19189c6cf4", "testCaseId": "dc0a9cd52b3898118b858a19189c6cf4", "fullName": "test.web.test_change_to_invalid_username.ProfileInvalidUsernameChange#test_change_to_invalid_username", "labels": [{"name": "feature", "value": "Username invalid - test"}, {"name": "story", "value": "Change to Invalid Username"}, {"name": "parentSuite", "value": "test.web"}, {"name": "suite", "value": "test_change_to_invalid_username"}, {"name": "subSuite", "value": "ProfileInvalidUsernameChange"}, {"name": "host", "value": "DESKTOP-OPJ94K2"}, {"name": "thread", "value": "24956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.web.test_change_to_invalid_username"}]}